### PR TITLE
Print cuSPARSELt version in `show_config`

### PR DESCRIPTION
**Pull request by asi1024**

### CODE REVIEW

File: cupy_backends/cuda/libs/cusparselt.pyx [hunk 1]
```diff
@@ -106,6 +106,9 @@ cdef extern from '../../cupy_cusparselt.h' nogil:
         const cusparseLtHandle_t* handle, const cusparseLtMatmulPlan_t* plan,
         const void* d_dense, void* d_compressed, driver.Stream stream)
 
+    # Version
+    int cusparseLtGetVersion(int* version)
```
Comment (kmaehashi) [inline cupy_backends/cuda/libs/cusparselt.pyx:110]: As talked offline, let's expose `int CUSPARSELT_VERSION` directly.

**Review by kmaehashi — CHANGES_REQUESTED** [review]

File: cupy_backends/cuda/libs/cusparselt.pyx (hunk 1 above)
Comment (leofang) [reply cupy_backends/cuda/libs/cusparselt.pyx:110]: Need to add `CUSPARSELT_VERSION` to the stub header.

File: cupyx/_runtime.py [hunk 2]
```diff
@@ -234,6 +243,7 @@ def __str__(self):
             ('NCCL Build Version', self.nccl_build_version),
             ('NCCL Runtime Version', self.nccl_runtime_version),
             ('cuTENSOR Version', self.cutensor_version),
+            ('cuSPARSELt Version', self.cusparselt_version),
```
Comment (kmaehashi) [inline cupyx/_runtime.py:246]:
```suggestion
            ('cuSPARSELt Build Version', self.cusparselt_version),
```

Comment (asi1024) [reply cupyx/_runtime.py]: Thanks!

**Review by kmaehashi — APPROVED** [review]

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
@kmaehashi Failed to backport automatically.

----

```
Not cleanly cherry-picked
```

**Comment by kmaehashi** [comment]
The above comment was to test https://github.com/cupy/backport/pull/28.

**Pull request merged**